inbm/dispatcher-agent/dispatcher/sota/os_factory.py

```diff
@@ -147,6 +152,10 @@ def create_snapshotter(self, sota_cmd: str, snap_num: Optional[str], proceed_wit
         trtl = Trtl(PseudoShellRunner(), BTRFS)
         return YoctoSnapshot(trtl, sota_cmd, self._dispatcher_callbacks, snap_num, proceed_without_rollback)
 
+    def delete_snapshot(self) -> int:
+        logger.debug("")
```

Comment:
It will be confusing later to have delete_snapshot do nothing for Yocto. I would say make delete_snapshot only for Ubuntu/Debian systems (don't have it implemented for the Yocto class), or implement the mender version here.